src/main/java/io/github/hapjava/characteristics/impl/thermostat/CurrentTemperatureCharacteristic.java

```diff
@@ -11,19 +11,29 @@
 public class CurrentTemperatureCharacteristic extends FloatCharacteristic {
 
   public CurrentTemperatureCharacteristic(
+      double minValue,
+      double maxValue,
+      double minStep,
       Supplier<CompletableFuture<Double>> getter,
       Consumer<HomekitCharacteristicChangeCallback> subscriber,
       Runnable unsubscriber) {
     super(
         "00000011-0000-1000-8000-0026BB765291",
         "current temperature",
-        0,
-        100,
-        0.1,
+        minValue,
+        maxValue,
+        minStep,
         "C",
         Optional.of(getter),
         Optional.empty(),
         Optional.of(subscriber),
         Optional.of(unsubscriber));
   }
+
+  public CurrentTemperatureCharacteristic(
+      Supplier<CompletableFuture<Double>> getter,
+      Consumer<HomekitCharacteristicChangeCallback> subscriber,
+      Runnable unsubscriber) {
+    this(0, 100, 0.1, getter, subscriber, unsubscriber);
```

Comment:
Wouldn't it be better  to have `public static final double MIN_TEMPERATURE_VALUE = 0;` (and similar)?`You could also re-use these in the Accessory

Reply:
very good idea. 
i would call the constant however DEFAULT_MIN_VALUE, DEFAULT_MAX_VALUE, DEFAULT_MINSTEP
so that we can use the same names for non-temperature related characteristics.  
and it is also clear that these are default not the actual values.

Reply:
Good idea.

Reply:
done